src/Coordination/KeeperStateMachine.h

```diff
@@ -123,7 +124,7 @@ class IKeeperStateMachine : public nuraft::state_machine
     /// In our state machine we always have a single snapshot which is stored
     /// in memory in compressed (serialized) format.
     SnapshotMetadataPtr latest_snapshot_meta = nullptr;
```

Comment:
Looks like `latest_snapshot_meta` and `latest_snapshot_buf` are also protected by `snapshots_lock` and should have TSA annotation

Reply:
Yep, thanks (this patch was just a "night hot fix" to the crash). Fixed